Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = "2021"
 keywords = ["api", "file", "network", "safe", "syscall"]
 categories = ["os::unix-apis", "date-and-time", "filesystem", "network-programming"]
 include = ["src", "build.rs", "Cargo.toml", "COPYRIGHT", "LICENSE*", "/*.md", "benches"]
-rust-version = "1.63"
```

Comment:
Could you revert this `rust-version` change? It isn't practical for us to bump the MSRV for all rustix users. Since wasip2 is a new target, I think we can just leave rust-version at 1.63 and just expect people using wasip2 to use the lastest Rust version.

Reply:
Sure.